### ISSUE TITLE
com.mi.globalbrowser (MIUI Browser) description might be wrong

### ISSUE THREAD

**Issue by Bqleine** [0x192/Universal-Android-Debloater#488]
**Your phone**: xiaomi redmi note 8T

**Packages documentation to update:**
```
com.mi.globalbrowser
```
## Documentation change
**List**: `Oem` (manufacturer).
**Removal**: `Recommended`

### Mi Browser
**Current description**
> [...]
> Note: Since MIUI 12 you can no longer uninstall this app. Disabling still works fine.

I have MIUI 12 and it uninstalled just fine, maybe this note should be removed or changed ?
Full version is MIUI Global 12.0.2 Stable 12.0.2.0(RCEUXM) on Android 11

**Comment by AnonymousWP** [comment]
Are you sure? Are you rooted? Because without root you can't, see: https://github.com/0x192/universal-android-debloater/wiki/FAQ#how-is-it-possible-to-delete-a-system-package-without-root-permission. Maybe the user back then uninstalled it using the built-in application manager in MIUI?

**Comment by Bqleine** [comment]
> Are you rooted?

No, and there is no "user back then" since it's my phone and it has been bought new, the application was present and usable before I uninstalled it.

> Are you sure?

There was no errors and the app looks gone, I don't know how to check if it's really really gone but it looks like it.
Looking in the UAD gui it now shows the "Restore" button like other uninstalled apps. The "disable" setting was not enabled.

I just selected the package in the list with the other ones even with the warnings just to try and it had no errors and then disappeared from the launcher.

It might be worth to note that it took me multiple times clicking on the uninstall button to clear the whole list, I had trouble connecting my phone so maybe the usb cable isn't working very good.

**Comment by Bqleine** [comment]
I used to have older MIUI versions, maybe the upgrade didn't care about making the app not uninstallable ?

**Comment by Bqleine** [comment]
Also, you mention an application manager in MIUI, I don't know of any such feature except the "System apps updater" which does not support uninstalling at all, in fact the uninstall button was disabled for the MI Browser.

**Comment by btvoidx** [comment]
Made a mistake of updating my Redmi Note 10, and now this monstrosity has invaded my app drawer. UAD does not appear to have it in any of the lists, including unsafe ones. It it safe to uninstall/disable?

**Comment by btvoidx** [comment]
Nvm. I restarted the phone and it allowed me to uninstall mi browser normally.
